README.md

```diff
@@ -137,6 +138,31 @@ Errors created with this package contain stack traces that are managed automatic
 }
 ```
 
+### Inverting the stack trace and error output
+
+If you prefer some other order than the default, `eris` supports inverting both the stack trace and the entire error output. When both are inverted, the root error is shown first and the original calling method is shown last.
+
+```golang
+// create a default format with error and stack inversion options
+format := eris.NewDefaultStringFormat(eris.FormatOptions{
+  InvertOutput: true,
+  WithTrace: true,
```

Comment:
suggest adding comments next to each flag in the example. I am not a big fan of this way to manipulate trace options because setting withTrace false renders InvertTrace useless but I cant think of a better and simpler approach rn

Reply:
done